UsacoLeeland/src/UsacoLeeland/BlockedBillboard.java

```diff
@@ -0,0 +1,114 @@
+package UsacoLeeland;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+public class BlockedBillboard 
+{
+	public static void sort(int z,int[][]x, String[]y)
+	{
+		for(int i=0; i<4; i++)
+		{
+			x[z][i]=Integer.parseInt(y[i]);
+		}
+	}
+	public static void main(String args[]) throws IOException
+	{
+		int dif=0;
+		int[][] cord=new int [3][4];
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		String temp=br.readLine();
+		sort(0,cord,temp.split("\\s+"));
+		temp=br.readLine();
+		sort(1,cord,temp.split("\\s+"));
+		temp=br.readLine();
+		sort(2,cord,temp.split("\\s+"));

```

Comment:
Learn to use "scanner". All USACO problems are better fit and simpler with scanner.